src/services/models/service.model.ts

```diff
@@ -0,0 +1,29 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql';
+import JSON from 'graphql-type-json';
+
+@ObjectType({ description: 'Services supported by the DampLab' })
+export class Service {
+  @Field(() => ID, { description: 'unique database generated ID' })
+  id: string;
+
+  @Field({ description: 'Human readable name of the service' })
+  name: string;
+
+  @Field({ description: 'URL to the icon of the service' })
+  icon: string;
+
+  @Field(() => JSON, { description: 'Parameters that are part of the service' })
+  parameters: any;
+
+  @Field(() => JSON, { description: 'Parameters defined earlier in the graph' })
+  flowParams?: any;
+
+  @Field(() => [String], { description: 'List of services this service can connect to' })
```

Comment:
Should this be a list of services?

Reply:
Yes it should be, for the hard coded testing I kept it as strings to match what Asad currently has stored and its easier to store just the strings in the hard coded array.